### ISSUE TITLE
View In Your Space on Editor not working?

### ISSUE THREAD

**Issue by octanebaby** [google/model-viewer#4271]
Using and iPhone 13 mini and an Phone 14 Pro:

Astronaut on the landing page: https://modelviewer.dev/ loads perfectly in AR

Astronaut (and all other models) on the Editor page: https://modelviewer.dev/editor/ does not.  Specifically, after deploying on mobile, the "View in Your Space" button renders but nothing happens when you click it.   (Regardless of whether or not this "Default AR Mode to Scene Viewer" box is checked). 

Other model-viewer apps working fine.

**Comment by niyouwoxi** [comment]
i have the same issue

**Comment by neostim** [comment]
Same here, iPhone 13 Pro Max, in Chrome, Brave and Safari, hitting the "View in Your Space" doesn't do anything.

**Comment by elalish** [comment]
@grorg was there some kind of Quick Look regression? Or maybe an incompatible USDz spec update?

**Comment by AEZ314** [comment]
Same. It works neither on my iPhone 14 pro max nor Samsung J8.

**Comment by AEZ314** [comment]
I found out that it works if you open a working quick look demo (on apple's site, for instance), reload the editor page on iPhone, and hit View In Your Space while it's still loading. I don't know. Maybe this helps.

**Comment by elalish** [comment]
@meshula Do you know of any Quick Look regressions in the last couple months?

For the rest of you, I'm not able to repro the problem on Android at least. Can anyone add some device specifics and if it's working elsewhere? It's exactly the same code running, so it would be odd if the AR mode was only broken in the editor's mobile view and no where else.

**Comment by meshula** [comment]
No, I haven't heard of anything that would explain this. One place to ask would be the ASWF USD working group on assets, there's folks there with knowledge of Quick Look issues and they might have some insight.

**Comment by ishook187** [comment]
I have the exact same issue. The HD Astronaut and *almost* all the other AR files work fine, but the QR code process would load a file 70% and then freeze. I tried a super simple object to make sure it wasn't textures or poly-count and it loaded via QR code but wouldn't go to AR. Latest IOS software and tried on an iphone XR and an older ipad. Same problems.

**Comment by Qumeric** [comment]
I also have this issue and I can confirm that it works only if "view in your space" is tapped before the model has finished loading which is sometimes tricky (it can be very fast).

**Comment by alrae** [comment]
I have the same issue on Iphone14 pro

**Comment by emeagenciadigital** [comment]
same issue

**Comment by pmorshed** [comment]
> Using and iPhone 13 mini and an Phone 14 Pro:
> 
> Astronaut on the landing page: https://modelviewer.dev/ loads perfectly in AR
> 
> Astronaut (and all other models) on the Editor page: https://modelviewer.dev/editor/ does not. Specifically, after deploying on mobile, the "View in Your Space" button renders but nothing happens when you click it. (Regardless of whether or not this "Default AR Mode to Scene Viewer" box is checked).
> 
> Other model-viewer apps working fine.

Have same problem (iPhone 13mini)

**Comment by elalish** [comment]
Okay, I have a repro on my very old iPhone, but not on my Pixel. I thought this was an iOS issue, but if someone has a repro on Samsung, then maybe not. I'll see if I can figure something out in Safari's dev tools.

**Comment by ConfidantCommunications** [comment]
On my iPhone 8, I can only see the astronaut which has the ios-src path specified. The auto-generated usdz doesn't work.

**Comment by ConfidantCommunications** [comment]
As I reported in another issue, the auto-generated usdz fails in Firefox for me but works in Safari.
